frontend/src/app/auth.service.ts

```diff
@@ -1,11 +1,11 @@
+import { HttpClient } from '@angular/common/http';
 import { Injectable } from '@angular/core';
-import { Http } from '@angular/http';
 import { Router } from '@angular/router';
 import { BehaviorSubject } from 'rxjs/BehaviorSubject';
 import { Observable } from 'rxjs/Observable';
 
 import { environment } from '../environments/environment';
-import { User } from './user.d';
+import { User } from './user';
```

Comment:
You have removed `.d` here but this file wasn't renamed?

Reply:
Good point. I will restore this line. It works like that BUT if somebody created a file  user.ts, then this import would fail.